src/main/java/uk/gov/hmcts/reform/hmi/runner/Runner.java

```diff
@@ -46,7 +46,9 @@ public void run(String... args) {
             //Process the selected blob
             //TODO Mocked for now
             String jsonData = processingService.processFile(blob);
-            distributionService.sendProcessedJson(jsonData);
+            if (jsonData != null) {
```

Comment:
We should check for if is null or empty. Can probably use one of the apache commons libs for it.

Reply:
agreed, that's a good way of doing it. C sharp has String.IsEmptyOrNull function for this. I used the function from library which you mentioned.